### DISCUSSION TITLE
About the use of Windows Dynamic Desktop software wallpaper refresh function, desktop background display problem

### DISCUSSION THREAD

**Discussion by cscowx** [t1m0thyj/WinDynamicDesktop#400] [Q&A]
Hi，

Windows Desktop software version is 4.7.0

Every time I refresh the wallpaper, it is always displayed as a tile. Is there any way to force the desktop wallpaper to be displayed as a fill?

**Comment by t1m0thyj** [comment]
Are you using the Microsoft Store version? If so, see the troubleshooting tip for [Wallpaper fit not saved with multiple monitors](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Known-issues#wallpaper-fit-not-saved-with-multiple-monitors).
